lib/vsc/utils/fancylogger.py

```diff
@@ -724,11 +724,13 @@ def _getSysLogFacility(name=None):
 
 def logToDevLog(enable=True, name=None, handler=None):
     """Log to syslog through /dev/log"""
```

Comment:
so this works without /dev/log? and does it work without th eoption when /dev/log is present?

Reply:
Yes, it now works without `/dev/log`. It'll just log to the default syslog location on the system when `/dev/log` (e.g. `/var/log/system/log` on my laptop).

Not sure I get your second question, which option do you mean?

Reply:
can you try on one of our hosts? i'm quite sure this is not always possible.
i guess your laptop runs some syslog service that listen on some default port and accepts anything?